app/views/display_task/index.html.erb

```diff
@@ -1,16 +1,21 @@
 <table class="table_data">
   <tr="table_headers">
     <th>Task Name</th>
+    <th>Person in Charge</th>
     <th>Completed?</th>
   </tr>
 
   <% @all_tasks.each_with_index do |task, index| %>
```

Comment:
Do you require access to the task's index, or could you just be using the `each`?